bin/runFixedPop

```diff
@@ -216,18 +216,24 @@ if __name__ == '__main__':
 
         # Select the initial binary sample method from user input
         if args.initial_samp == 'independent':
-            IBT, sampled_mass, n_samp = InitialBinaryTable.sampler(args.initial_samp, kstar1_range, kstar2_range, 'kroupa93', 'thermal', SFH_model, component_age, args.metallicity, args.Nstep)
+            IBT, mass_singles, mass_binaries, n_singles, n_binaries = InitialBinaryTable.sampler(args.initial_samp, kstar1_range, kstar2_range, 0.5, 'kroupa93', 'thermal', SFH_model, component_age, args.metallicity, args.Nstep)
 
         if args.initial_samp == 'multidim':
-            IBT, sampled_mass, n_samp = InitialBinaryTable.sampler(args.initial_samp, kstar1_range, kstar2_range, rand_seed, args.nproc, SFH_model, component_age, args.metallicity, args.Nstep)
+            IBT, mass_singles, mass_binaries, n_singles, n_binaries= InitialBinaryTable.sampler(args.initial_samp, kstar1_range, kstar2_range, rand_seed, args.nproc, SFH_model, component_age, args.metallicity, args.Nstep)
 
         # Log the total sampled mass from the initial binary sample
         # for future Galactic occurence rate calculation
-        total_sampled_mass += sampled_mass
+        total_mass_singles += mass_singles
+        total_mass_binaries += mass_binaries
+        total_n_singles += n_singles
+        total_n_binaries += n_binaries
 
         # save the total_sampled_mass so far
-        dat_store.append('totalMass', pd.DataFrame([total_sampled_mass]))
-        log_file.write("The total mass sampled so far is: {0}\n".format(total_sampled_mass))
+        dat_store.append('mass_singles', pd.DataFrame([total_mass_singles]))
+        dat_store.append('mass_binaries', pd.DataFrame([total_mass_binaries]))
+        dat_store.append('n_singles', pd.DataFrame([total_n_singles]))
+        dat_store.append('n_binaries', pd.DataFrame([total_n_binaries]))
```

Comment:
Through redundant, I think it would be useful to store the total mass and total number of stars (n_singles + 2*n_binaries) as well.